File: HRBackend/HRBackend/Models/Empleado/Empleado.cs
using HRBackend.Models.Candidatos;

namespace HRBackend.Models.Empleado
{
    public class Empleado //: Usuario
    {
        public int  Id { get; set; }
        public string Cedula { get; set; }
        public string Nombre { get; set; }
        public DateTime FechaIngreso { get; set; }
        public string Departamento { get; set; }
        public string Puesto { get; set; }
        public decimal SalarioMensual { get; set; }
        public bool Estado { get; set; } // Activo, Inactivo, etc.
        //public int? UsuarioId { get; set; } = 0;
        //public virtual Usuario? Usuario { get; set; } = null;
    }
}

File: frontend_angular/src/app/models/class/Empleado.ts
export class Empleado{
id: number;
cedula: string;
nombre: string;
fechaIngreso: string;
departamento: string;
puesto: string;
salarioMensual: number;
estado: boolean;

constructor(){
    this.id = 0,
    this.cedula = '',
    this.nombre = '',
    this.departamento = '',
    this.fechaIngreso = new Date().toISOString().split('T')[0],
    this.puesto = '',
    this.salarioMensual = 0,
    this.estado = true
}
}
